protonvpn_cli/utils.py

```diff
@@ -11,8 +11,8 @@
 import random
 # External Libraries
 import requests
-from .logger import logger
 # Constants
+from .logger import logger
```

Comment:
Strictly speaking that is not a constant

Reply:
Looks like I wasn't paying attention...